adminer/include/editing.inc.php

```diff
@@ -290,7 +290,6 @@ function edit_fields($fields, $collations, $type = "TABLE", $foreign_keys = arra
 * @return bool
 */
 function process_fields(&$fields) {
-	ksort($fields);
```

Comment:
Thanks for the research, I'm glad that we could lift this limitation. I don't remember what made me think that the order could be random, maybe some obscure browser.